addon/core/rdfa-processor.ts

```diff
@@ -112,10 +168,13 @@ export function preprocessRDFa(dom: Node) {
   }
   // each content node
   for (const [node, object] of datastore.getContentNodeMap().entries()) {
-    const { subject, predicate } = object;
-    const incomingProp: Backlink = {
+    const { subject, predicate, datatype, language } = object;
+    const incomingProp: IncomingTriple = {
+      termType: 'LiteralNode',
       subject: subject.value,
       predicate: predicate.value,
+      datatype: datatype ?? { termType: 'NamedNode', value: '' },
+      language: language || '',
```

Comment:
We probably need to change this to
```suggestion
      language: datatype ? '' : language ?? '',
```